ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/DTOs/CreateBulkTaskDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.DTOs
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class CreateBulkTaskDTO
```

Comment:
BulkCreateTaskDTO, so it's clear from the first view that it's especially Bulk (common naming convention)